test/mocks/router/mocks.h

```diff
@@ -397,8 +382,7 @@ class MockRouteConfigProvider : public RouteConfigProvider {
 
 class MockRouteConfigProviderManager : public RouteConfigProviderManager {
 public:
-  MockRouteConfigProviderManager();
-  ~MockRouteConfigProviderManager() override;
+  MockRouteConfigProviderManager() = default;
```

Comment:
Likewise, constructors are only declared in the header file and defined in the .cc to improve compilation performance.

Reply:
done